lib/oli/publishing.ex

```diff
@@ -1126,4 +1130,19 @@ defmodule Oli.Publishing do
 
     Enum.map(results, fn [slug, title] -> %{slug: slug, title: title} end)
   end
+
+  @doc """
+    Refreshes the part_mapping materialized view.
+    Since this operation is expensive, do not use perform it synchronously unless neccesary.
+  """
+  def refresh_part_mapping() do
+    Repo.query("REFRESH MATERIALIZED VIEW CONCURRENTLY part_mapping")
+    Logger.info("Refreshed part_mapping view")
```

Comment:
I'm not sure we need to log this - seems like extra noise, WDYT?

Reply:
No strong opinion - used it for debugging mostly and figured we could keep it to see the periodicity with which this is being run in prod. Now, having AppSignal setup we can totally remove this.